ansible/playbooks/hosts

```diff
@@ -47,12 +47,9 @@ apollo-monitor.genome.edu.au
 apollo-portal.genome.edu.au
 #apollo-portal-main.genome.edu.au # use this when apollo-portal redirects to apollo-portal-offline
 apollo-deploy.genome.edu.au
-ubuntu22-test.genome.edu.au
```

Comment:
can you put ubuntu22-test.genome.edu.au back in the hosts inventory file. ie just revert ansible/playbooks/hosts  to what it was before you made any changes (ie it shouldn't be included in this PR). otherwise I need to add it back for system updates.